docker/artifactory_setup

```diff
@@ -0,0 +1,6 @@
+export BUNDLE_EZCATER__JFROG__IO=${BUNDLE_EZCATER__JFROG__IO:-$(grep -e "^BUNDLE_EZCATER__JFROG__IO:" ~/.bundle/config | cut -d\" -f 2)}
```

Comment:
I recommend dropping this file as we don't use private gems for this open source project.